lib/Models/Catalog/CatalogReferences/SplitItemReference.ts

```diff
@@ -25,20 +28,40 @@ export default class SplitItemReference extends ReferenceMixin(
     previousTarget: BaseModel | undefined
   ): Promise<BaseModel | undefined> {
     if (this.splitSourceItemId === undefined || this.uniqueId === undefined) {
-      return;
+      throw new TerriaError({
+        title: i18next.t("splitterTool.errorTitle"),
```

Comment:
We should prefer `{key: "splitterTool.errorTitle"}` over translating inside lib/Models.